nautobot/dcim/views.py

```diff
@@ -4168,6 +4171,100 @@ class PowerFeedUIViewSet(NautobotUIViewSet):
     serializer_class = serializers.PowerFeedSerializer
     table_class = tables.PowerFeedTable
 
+    @staticmethod
+    def _render_utilization_graph(value):
+        if not value or not isinstance(value, tuple) or len(value) != 2:
+            return helpers.placeholder(None)
+        allocated, available = value
+        if available <= 0:
+            return f"{allocated}VA / {available}VA"
+        graph_html = render_to_string(
+            "utilities/templatetags/utilization_graph.html",
+            helpers.utilization_graph_raw_data(allocated, available),
+        )
+        return format_html("{}VA / {}VA {}", allocated, available, graph_html)
+
+    object_detail_content = object_detail.ObjectDetailContent(
+        panels=(
+            object_detail.KeyValueTablePanel(
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                label="Power Feed",
+                context_data_key="powerfeed_data",
+                value_transforms={
+                    "Power Panel": [helpers.hyperlinked_object],
+                    "Rack": [helpers.hyperlinked_object],
+                    "Type": [helpers.placeholder],
+                    "Status": [helpers.hyperlinked_object_with_color],
+                    "Connected Device": [helpers.mark_safe],
```

Comment:
Are these needed? I would have expected these five to render correctly by default.